pkg/config/usagereportingconfig.go

```diff
@@ -54,12 +46,10 @@ type UsageReportingConfig interface {
 // UsageReportingConfiguration - structure to hold all usage reporting settings
 type UsageReportingConfiguration struct {
 	UsageReportingConfig
-	Publish           bool          `config:"publish"`
-	PublishMetric     bool          `config:"publishMetric"`
-	Interval          time.Duration `config:"interval"`
-	UsageSchedule     string        `config:"usageSchedule"`
-	Offline           bool          `config:"offline"`
-	Schedule          string        `config:"offlineSchedule"`
+	Publish           bool   `config:"publish"`
+	UsageSchedule     string `config:"usageSchedule"`
```

Comment:
I don't care either way, but maybe we can migrate this to just schedule? So env var is `CENTRAL_USAGEREPORTING_SCHEDULE`